docs/content/en/docs/clustermgmt/cluster-backup-restore/restore-cluster.md

```diff
@@ -0,0 +1,245 @@
+
+    ```bash
+    kubectl get nodes --kubeconfig $WORKLOAD_CLUSTER_2_KUBECONFIG
+
+    NAME                              STATUS     ROLES           AGE     VERSION
+    w01-bbtdd                         NotReady   control-plane   3d23h   v1.27.3-eks-6f07bbc
+    w01-md-0-66dbcfb56cxng8lc-8ppv5   NotReady   <none>          3d23h   v1.27.3-eks-6f07bbc
+    ```
+
+    ```bash
+    kubectl describe node w01-bbtdd --kubeconfig $WORKLOAD_CLUSTER_2_KUBECONFIG
+
+    Name:               w01-bbtdd
+    ...
+    Conditions:
+      Type             Status    LastHeartbeatTime                 LastTransitionTime                Reason              Message
+      ----             ------    -----------------                 ------------------                ------              -------
+      MemoryPressure   Unknown   Mon, 09 Oct 2023 21:55:58 +0000   Mon, 09 Oct 2023 22:34:40 +0000   NodeStatusUnknown   Kubelet stopped posting node status.
+      DiskPressure     Unknown   Mon, 09 Oct 2023 21:55:58 +0000   Mon, 09 Oct 2023 22:34:40 +0000   NodeStatusUnknown   Kubelet stopped posting node status.
+      PIDPressure      Unknown   Mon, 09 Oct 2023 21:55:58 +0000   Mon, 09 Oct 2023 22:34:40 +0000   NodeStatusUnknown   Kubelet stopped posting node status.
+      Ready            Unknown   Mon, 09 Oct 2023 21:55:58 +0000   Mon, 09 Oct 2023 22:34:40 +0000   NodeStatusUnknown   Kubelet stopped posting node status.
+    Events:
+      Type    Reason          Age    From             Message
+      ----    ------          ----   ----             -------
+      Normal  RegisteredNode  9m32s  node-controller  Node w01-bbtdd event: Registered Node w01-bbtdd in Controller
+      Normal  NodeNotReady    8m52s  node-controller  Node w01-bbtdd status is now: NodeNotReady
+    ```
+
+1. Restart Kubelet of the control plane and worker nodes of the workload cluster `w02`.
+
+    In order to bring back the nodes to ready state, you need to manually restart the Kubelet on all the control plane and worker nodes. Kubelet registers the node itself with the apisever which then updates etcd with the correct node data of the new workload cluster `w02`.
+
+    {{< tabpane >}}
+    {{< tab header="Ubuntu or RHEL" lang="bash" >}}
+# SSH into the control plane and worker nodes. You must do this for each node.
+ssh -i ${SSH_KEY} ${SSH_USERNAME}@<node IP>
+sudo su
+systemctl restart kubelet
+    {{< /tab >}}
+    {{< tab header="Bottlerocket" lang="bash" >}}
+# SSH into the control plane and worker nodes. You must do this for each node.
+ssh -i ${SSH_KEY} ${SSH_USERNAME}@<node IP>
+apiclient exec admin bash
+sheltie
```

Comment:
```suggestion
sudo sheltie
```
Or is the sudo not required here? I think if you ssh to the node and directly run sudo sheltie, that is also enough not requiring you to run the apiclient command right?

Reply:
when you are in admin container, sudo is not required